docs/api.md

```diff
@@ -159,23 +158,23 @@ const normalizedData = normalize(data, myArray);
 
 ### `Entity(key, definition = {}, options = {})`
 
-* `key`: **required** The key name under which all entities of this type will be listed in the normalized response. Must be a string name.
+* `key`: **required** The key name under which all entities of this type will be listed in the normalized response. Must be a string name or [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol).
```

Comment:
only actual change; rest is prettier autorun

Reply:
For the future, can you please configure your editor to not change files that you haven't modified? prettier within this repo is only run on files that have been modified.